cmd/crates/stellar-ledger/src/speculos.rs

```diff
@@ -0,0 +1,75 @@
+use std::{collections::HashMap, path::PathBuf};
```

Comment:
This file is focused on setting up the Ledger emulator, Speculos, to be run as a testcontainer.